src/main/java/net/mcreator/ui/modgui/ModElementGUI.java

```diff
@@ -331,7 +331,7 @@ protected final void finalizeGUI(boolean wrapInScrollPane) {
 				splitScroll.getHorizontalScrollBar().setUnitIncrement(15);
 				parameters = new JLayer<>(splitScroll, new ScrollWheelPassLayer());
 			} else {
-				parameters = PanelUtils.join(split);
```

Comment:
This change is not related to this PR.

Reply:
As you can see in the files tab, it was moved to the master before this comment was posted. 

![image](https://github.com/MCreator/MCreator/assets/16374228/ba8fdb80-d3c1-4179-8260-0902df614d5d)

Please F5 the page if you still see the change